src/utils.ts

```diff
@@ -25,12 +25,16 @@ import {
     generateKeyPairFromSeed,
     KeyPair
 } from '@stablelib/ed25519'
-import { sha256 } from '@cosmjs/crypto'
+import { DirectSecp256k1HdWallet, DirectSecp256k1Wallet } from '@cosmjs/proto-signing'
+import { EnglishMnemonic as _, sha256 } from '@cosmjs/crypto'
 import { v4 } from 'uuid'
 import {
     VerificationMethod as ProtoVerificationMethod,
     Service as ProtoService,
-} from "@cheqd/ts-proto/cheqd/did/v2/index.js"
+    MsgCreateDidDocPayload,
```

Comment:
Do we need to support MsgUpdateDidDocPayload and MsgCreateResourcePayload also?

Reply:
Create did and update did have the same payload in sdk, Was not sure if we should create separate functions even if they are the same one's

Reply:
I think it would be nice to have such function even if it's the same inside but it's more clear for understanding